Source/Core/Core/PowerPC/SignatureDB.cpp

```diff
@@ -2,74 +2,42 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#include <fstream>
+#include <ios>
+#include <sstream>
 #include <string>
 
 #include "Common/CommonTypes.h"
 #include "Common/FileUtil.h"
 #include "Common/Logging/Log.h"
+#include "Common/StringUtil.h"
 #include "Core/PowerPC/PPCAnalyst.h"
 #include "Core/PowerPC/PPCSymbolDB.h"
 #include "Core/PowerPC/PowerPC.h"
 #include "Core/PowerPC/SignatureDB.h"
 
-namespace
-{
-// On-disk format for SignatureDB entries.
-struct FuncDesc
-{
-  u32 checkSum;
-  u32 size;
-  char name[128];
-};
+// Format Handlers
+#include "Core/PowerPC/CSVSignatureDB.h"
```

Comment:
SignatureDB-related stuff should probably be moved to its own subdirectory, otherwise this sort of clutters up the PowerPC directory.